bedrock/settings/base.py

```diff
@@ -1797,3 +1856,98 @@ def before_send(event, hint):
 RELAY_PRODUCT_URL = config(
     "RELAY_PRODUCT_URL", default="https://stage.fxprivaterelay.nonprod.cloudops.mozgcp.net/" if DEV else "https://relay.firefox.com/"
 )
+
+# WAGTAIL =======================================================================================
+
+WAGTAIL_SITE_NAME = config(
+    "WAGTAIL_SITE_NAME",
+    default="Mozorg (selective)",
+)
+
+WAGTAILADMIN_BASE_URL = config(
+    "WAGTAILADMIN_BASE_URL",
+    default="",
+)
+
+# We're sticking to LTS releases of Wagtail, so we don't want to be told there's a new version if that's not LTS
+WAGTAIL_ENABLE_UPDATE_CHECK = False
+
+# Custom setting (not a Wagtail core one) that we use to plug in/unplug the admin UI entirely
+WAGTAIL_ENABLE_ADMIN = config(
+    "WAGTAIL_ENABLE_ADMIN",
+    default="False",
+    parser=bool,
+)
+
+if WAGTAIL_ENABLE_ADMIN:
```

Comment:
While all Wagtail-managed pages will be in the DB and therefore accessible all the time, we are making the ability to expose the Wagtail editing UI configurable. This is part of the bigger plan to have a secured "editing node"